src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx

```diff
@@ -225,12 +225,11 @@ export const ChooseRightsPage = () => {
                 {t('common.confirm')}
               </Button>
             </div>
-          </Popover.Content>
-        </Popover>
+          </Popover>
+        </Popover.Context>
         <Button
           variant='secondary'
-          color='first'
-          fullWidth={isSm}
```

Comment:
I'm guessing `fullWidth` is not a thing anymore?
Should we then add some manual styling to make the buttons take up all horizontal space on small screens?
![image](https://github.com/user-attachments/assets/2de61943-caa8-4e10-a1ce-7c39beb4bf64)